### ISSUE TITLE
what's the default activation function of cudnnlstm in tensorflow? How can I set an activation function such as relu?

### ISSUE THREAD

**Issue by fudanchenjiahao** [tensorflow/tensorflow#23297]
what's the default activation function of cudnnlstm in tensorflow? How can I set an activation function such as relu? Maybe it's just linear model? I read the document, but I did not find it.
for example, the code is below:

lstmcell=tf.contrib.cudnn_rnn.CudnnLSTM(1,encoder_size,direction="bidirectional")

hq,_ =lstmcell(query)

and i read the document in tensorflow https://www.tensorflow.org/api_docs/python/tf/contrib/cudnn_rnn/CudnnLSTM?hl=zh-cn the functon is below

init(

num_layers,
num_units,
input_mode=CUDNN_INPUT_LINEAR_MODE,
direction=CUDNN_RNN_UNIDIRECTION,
dropout=0.0,
seed=None,
dtype=tf.float32,
kernel_initializer=None,
bias_initializer=None,
name=None
)

And no keyword to set a parameter such as "activation = "tanh" just like tf.nn.rnn_cell.LSTMell.

So what's the default activation function of cudnnlstm in tensorflow, and how to change it to leaky_relu.

**Comment by harshini-gadige** [comment]
You may refer [this.](https://github.com/keras-team/keras/issues/8510)


These type of questions are better asked on [StackOverflow](http://stackoverflow.com/questions/tagged//tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.
